vehicle/vehicle_info_util/include/vehicle_info_util/vehicle_info.hpp

```diff
@@ -44,6 +46,35 @@ struct VehicleInfo
   double max_lateral_offset_m;
   double min_height_offset_m;
   double max_height_offset_m;
+
+  tier4_autoware_utils::LinearRing2d createFootprint(const double margin = 0.0) const
```

Comment:
There is no problem, but just a question.
Why did you create a wrapper function instead of settings default values to lat/lon margins like this?

```
const double lat_margin = 0.0, const double lon_margin = 0.0
```

Reply:
It seems only lane_departure_checker uses respective margin for lateral and longitudinal, and others use the same margin for lateral and longitudinal.
Also, in most cases, the margin is set 0.